nodemapper/src/gui/Builder/components/ParameterList.tsx

```diff
@@ -0,0 +1,476 @@
+                  style={{ maxHeight: "25px" }}
+                  onClick={disconnectParameter}
+                >
+                  Disconnect
+                </Button>
+              </TreeItem>
+            );
+          } else {
+            return (
+              <TreeItem
+                key={node.id + "__" + identifier}
+                nodeId={(nodeId++).toString()}
+                label={key + ": " + value}
+                onClick={() => onParameterSelect(node, keylist, key)}
+              />
+            );
+          }
+        } else {
+          return (
+            <TreeItem
+              style={{ color: "#bebebe" }}
+              key={node.id + "__" + identifier}
+              nodeId={(nodeId++).toString()}
+              label={key + ": " + value}
+            />
+          );
+        }
+      };
+
+      return (
+        <>
+          {isSimpleValue ? (
+            <DisplaySimpleValue />
+          ) : (
+            <TreeItem
+              key={node.id + "__" + [...keylist, key].join("/")}
+              nodeId={(nodeId++).toString()}
+              label={label}
+            >
+              <NodeParameters node={node} keylist={[...keylist, key]} />
+            </TreeItem>
+          )}
+        </>
+      );
+    });
+    return <>{fieldlist}</>;
+  };
+
+  const ModuleEntry = ({ label, node }: IModuleEntryProps) => (
+    <Item>
+      <div
+        key={node.id}
+        style={{
+          display: "flex",
+          flexDirection: "column",
+          gap: "10px",
+        }}
+      >
+        <div
+          style={{
+            display: "flex",
+            flexDirection: "column",
+            gap: "0px",
+            backgroundColor: "#eee",
+          }}
+        >
+          {showAllNodes || showSelfNodes ? (
+            <div>{"Node: " + label}</div>
+          ) : (
+            <div>
+              {"Port: " + label}
+              <div>
+                <small>{node.data.config.name}</small>
+              </div>
+            </div>
+          )}
+        </div>
+        <div
+          key={node.id + "_p"}
+          style={{
+            display: "flex",
+            flexDirection: "column",
+            gap: "3px",
+          }}
+        >
+          <TreeView
+            defaultExpanded={Array.from({ length: 999 }, (_, i) =>
+              i.toString(),
+            )}
+          >
+            <NodeParameters node={node} keylist={[]} />
+          </TreeView>
+        </div>
+      </div>
+    </Item>
+  );
+
+  // Handle escape key
+  document.onkeydown = (e: KeyboardEvent) => {
+    if (e.key === "Escape") {
+      onclose();
+    }
+  };
+
+  return (
+    <div
+      style={{ top, left, right, bottom, margin: "5px" }}
+      className="parameter-list"
+      {...props}
+    >
+      <p style={{ margin: "0.5em" }}>
+        <big>
+          <b>{keylist_str}</b>
+        </big>{" "}
+        <i>{node_name}</i>
+        <span style={{ float: "right" }}>
+          {isConnected ? (
```

Comment:
I think this can be just:
```suggestion
          {isConnected && (
```
and the ` : null` can be dropped at the end of this block.